target_bigquery.py

```diff
@@ -132,8 +132,9 @@ def persist_lines_job(project_id, dataset_id, lines=None):
 
             validate(msg.record, schema)
 
-            dat = bytes(str(json.loads(json.dumps(msg.record), object_pairs_hook=clear_dict_hook)) + '\n', 'UTF-8')
-            
+            # NEWLINE_DELIMITED_JSON expects literal JSON formatted data, with a newline character splitting each row.
+            dat = bytes(json.dumps(msg.record) + '\n', 'UTF-8')
```

Comment:
I'm not sure how this code was working before; perhaps the old client version that you were using didn't actually use JSON format? 

The old code was outputting python dict-format strings, which BigQuery doesn't support.

Did this ever work? Or was this perhaps a change between the client versions?